makefile

```diff
@@ -11,6 +11,7 @@ packages += rest-client rest-data rest-route rest-model migrations trigger model
 lintOnlyPackages := testutil model-manifest
 orgPath := github.com/evergreen-ci
 projectPath := $(orgPath)/$(name)
+lobsterTempDir := $(shell pwd)/$(buildDir)/.lobster-temp
```

Comment:
If you're going to use `$(shell pwd)` everywhere, you might as well just `export EVGHOME := $(shell pwd)` rather than copy paste the same `EVGHOME=$(shell pwd)`. That would remove the need to set EVGHOME in several different places.

Also, is there any reason that it's a dotfile directory? Why not just make it a regular directory?

Reply:
The example I was referring to in the Logkeeper repo stored these files within a dotfile directory and I think the reason was to just hide the files. I'll remove the dot